draft-ietf-quic-multipath.md

```diff
@@ -522,31 +514,46 @@ for subsequent connections, it might be difficult for the client
 to pick the "right" token among multiple tokens obtained in a previous connection.
 The client is likely to fall back to the strategy specified in {{Section 8.1.3 of QUIC-TRANSPORT}},
 i.e., pick the last received token. To avoid issues when clients make the "wrong" choice,
-a server should issue a token that is capable of validating
+a server SHOULD issue tokens that are capable of validating
 any of the previously validated addresses. Further guidance on token usage can be
 found in {{Section 8.1.3 of QUIC-TRANSPORT}}.
 
 ## Handling Connection IDs {#consume-retire-cid}
 
-With the multipath extension, each connection ID is associated with one path
-that is identified by the Path ID that is specified in the Path Identifier field of
-the PATH_NEW_CONNECTION_ID frame {{mp-new-conn-id-frame}}.
-The Path ID 0 indicates the initial path of the connection.
-Respectively, the connection IDs used during the handshake belong to the initial path
-with Path ID 0.
+In order to identify a connection ID correctly when the multipath extension is used,
+endpoints have to use the PATH_NEW_CONNECTION_ID and PATH_RETIRE_CONNECTION_ID frames
+instead of the NEW_CONNECTION_ID and RETIRE_CONNECTION_ID frame to indicate
+the respective Path ID together with associated sequence number
+(see {{Section 5.1.1 of QUIC-TRANSPORT}}), at least for all paths with a Path ID other than 0.
+Each Path ID has its own connection ID sequence number space whose initial value is 0.
+
+Endpoints can also use PATH_NEW_CONNECTION_ID and
+PATH_RETIRE_CONNECTION_ID for the initial path with Path ID 0,
+however, the use of NEW_CONNECTION_ID and RETIRE_CONNECTION_ID
+is still valid and endpoints need to process these frames accordingly
+as corresponding to Path ID 0.
```

Comment:
Note regarding other editorial passes: we should then remove redundant text in the frame section. We may keep this text here, but I think merging it with Section 2.3 would avoid repeating this for all the PATH_* frames.

Reply:
I don't think the Frames section says this...? Which section do you mean by 2.3?

Reply:
@qdeconinck what do you mean?

Reply:
I guess @qdeconinck is referring to [Section 7.4](https://datatracker.ietf.org/doc/html/draft-ietf-quic-multipath-13#name-path_new_connection_id-fram):

> Note that the NEW_CONNECTION_ID frame can only be used to issue or retire connection IDs for the initial path with Path ID 0.

Reply:
Yeah, that was what @Yanmei-Liu said. What I meant is that we should say at one place that NEW_CONNECTION_ID, RETIRE_CONNECTION_ID and ACK frames are still valid and are equivalent to their PATH_* versions with Path ID 0. We can delegate this to a further issue, but we can just merge this for now.